setup.py

```diff
@@ -1,15 +1,21 @@
-from distutils.core import setup
-import subconscious
+#!/usr/bin/env python3
+
+from setuptools import setup, find_packages
+
 
 setup(
     name='subconscious',
-    version=subconscious.__version__,
-    packages=['subconscious',
-              ],
-    url='',
-    license='',
-    author='paxosdev',
-    author_email='',
-    description='',
-    install_requires=['aioredis', 'python>=3.6']
```

Comment:
Turns out this wasn't supported. I don't know why though, `python<3` works for enforcing python2!